### PR TITLE
chore: set the plugin visibility

### PR DESCRIPTION
**Pull request by FeiWang1119**
set the plugin in tray visibility

Log:

### CODE REVIEW

File: panels/dock/tray/CMakeLists.txt [hunk 1]
```diff
@@ -26,4 +26,18 @@ target_link_libraries(dock-tray PUBLIC
...
+    trayitem.h
+    ../dockiteminfo.cpp
+    ../dockiteminfo.h
+)
+
+target_link_libraries(trayitem PRIVATE
+    dde-shell-frame
+)
+
+ds_install_package(PACKAGE org.deepin.ds.dock.trayItem TARGET trayitem)
```
Comment (tsic404) [inline panels/dock/tray/CMakeLists.txt:42]: 和上面的tray合并到一起

File: panels/dock/tray/trayitem.cpp [hunk 2]
```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: 2024 UnionTech Software Technology Co., Ltd.
+//
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+#include "trayitem.h"
+
+#include "panels/dock/constants.h"
+#include "panels/dock/dockiteminfo.h"
```
Comment (tsic404) [inline panels/dock/tray/trayitem.cpp:8]: set_include_path 吧

File: panels/dock/tray/package/tray.qml [hunk 3]
```diff
@@ -22,6 +22,9 @@ AppletItem {
     readonly property var filterTrayPlugins: ["dde-quick-panel","sound"]
     implicitWidth: useColumnLayout ? Panel.rootObject.dockSize : overflowId.implicitWidth
     implicitHeight: useColumnLayout ? overflowId.implicitHeight : Panel.rootObject.dockSize
+    Component.onCompleted: {
```
Comment (tsic404) [inline panels/dock/tray/package/tray.qml:25]: 用绑定

File: panels/dock/tray/trayitem.cpp (hunk 2 above)
Comment (FeiWang1119) [reply panels/dock/tray/trayitem.cpp:8]: done

File: panels/dock/tray/package/tray.qml (hunk 3 above)
Comment (FeiWang1119) [reply panels/dock/tray/package/tray.qml:25]: done

**Pull request closed without merging**